api/api_unix.go

```diff
@@ -7,8 +7,8 @@
 
 package api
 
-// #cgo darwin LDFLAGS: -lodbc -L/opt/local/lib
-// #cgo darwin CFLAGS: -I/opt/local/include
+// #cgo darwin LDFLAGS: -lodbc -L/usr/local/lib
```

Comment:
I do not have Darwin OS myself. This line was written by someone else who had Darwin OS too. How did that work on his computer?

Perhaps you could change darwin LDFLAGS and CFLAGS lines to match what linux does - not to include -I and -L flags altogether? Does that work for you?

Reply:
yes, when i change path to /usr , Now it's ok !

Reply:
Perhaps, I did not explain myself properly. If you remove

// #cgo darwin LDFLAGS: -lodbc -L/opt/local/lib
// #cgo darwin CFLAGS: -I/opt/local/include

and replace these 2 lines with single line

// #cgo darwin LDFLAGS: -lodbc

. Does your program still builds?

Reply:
> so I think not need add the args "-L/opt/local/lib" and only need single line

If you change your PR accordingly, I will merge it.

Thank you.

Alex

Reply:
I changed the PR, pls review it.